src/main/java/org/com/itpple/spot/server/controller/UserInfoController.java

```diff
@@ -43,4 +38,26 @@ public ResponseEntity<Void> nicknameCheck(@RequestParam("nickname") String nickn
         return ResponseEntity.ok().build();
     }
 
+    //유저 조회
+    @GetMapping("/info")
+    public ResponseEntity<UserInfoResponse> getUserInfo(@Auth CustomUserDetails customUserDetails) {
+        Long userId = customUserDetails.getUserId();
+        return ResponseEntity
+                .status(HttpStatus.CREATED)
+                .body(userInfoService.getUserInfo(userId));
+    }
+
+    //유저 정보 수정
+    @PutMapping("/update")
```

Comment:
마찬가지로 이 부분도 이렇게 작성하면 좋을 거 같아요!
```suggestion
    @PutMapping("")
```

Reply:
수정했습니다!